src/test/java/RecipeTest.java

```diff
@@ -35,17 +40,15 @@
 
   @Before
   public void setup() {
-    recipe = new Recipe(NAME, DESCRIPTION, new LinkedList(STEPS));
+    recipe = new Recipe(NAME, DESCRIPTION, TAGS, INGREDIENTS, new LinkedList(STEPS));
   }
 
   @Test
   public void appendStep() {
     List<Step> expectedSteps = new LinkedList<>(STEPS);
     expectedSteps.add(new Step("butter the bread"));
-
-    Recipe expected = new Recipe(NAME, DESCRIPTION, expectedSteps);
+    Recipe expected = new Recipe(NAME, DESCRIPTION, TAGS, INGREDIENTS, expectedSteps);
```

Comment:
Like I mentioned in a previous PR, you can do the following
expectedSteps = steps + new step.
recipe.appendStep()
Assert.assertEquals(expectedSteps, recipe.getSteps())

This is a much better practice as it tests exactly for what is to be tested.
It also avoids creating the unnecessary expected recipe object.

Reply:
Please make such a change either in this PR or another and submit it before this one.

Reply:
This has since been resolved in a PR that has already merged: PR #29

Reply:
The merge conflicts I just resolved should reflect the correct tests as well.